components/admin/test-suite/SPECS/tests.spec

```diff
@@ -8,73 +8,77 @@
 #
 #----------------------------------------------------------------------------eh-
 
+%global __brp_mangle_shebangs_exclude bats
+%undefine __brp_mangle_shebangs
+
 %include %{_sourcedir}/OHPC_macros
 
+%define testuser ohpc-test
+%define homedir %(sed -n "s/^\s*HOME=\s*//p" /etc/default/useradd || echo "/home" )
+
 Summary:   Integration test suite for OpenHPC
 Name:      test-suite%{PROJ_DELIM}
-Version:   2.6.0
+Version:   3.0
 Release:   1
 License:   Apache-2.0
 Group:     %{PROJ_NAME}/admin
 BuildArch: noarch
 URL:       https://github.com/openhpc/ohpc/tests
 Source0:   tests-ohpc.tar
+# In the OHPC source root dir, create with "tar -cf tests-ohpc.tar tests/"
 
 BuildRequires:  autoconf%{PROJ_DELIM}
 BuildRequires:  automake%{PROJ_DELIM}
+BuildRequires:  sed, findutils
 
-%global __brp_mangle_shebangs_exclude bats
-%undefine __brp_mangle_shebangs
-
-%if 0%{?suse_version}
+%if 0%{?suse_version} || 0%{?sle_version}
```

Comment:
From what I saw in `rpm --showrc` checking for `%{?sle_version}` should be enough I think.

Reply:
I think you're correct, but I'm going by the chart here: 
https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto
It implies that someone running Tumbleweed requires a suse_version check. I know it's not formally supported, but I still want it to work.
I've been kicking around adding %{suse} to OHPC_MACROS as the counterpart to %{rhel}.